src/index.ts

```diff
@@ -12,9 +12,11 @@ export {AppBridgeContext} from './utilities/app-bridge';
 
 /* eslint-disable @typescript-eslint/camelcase */
 export {
+  buildCustomProperties as UNSTABLE_buildCustomProperties,
```

Comment:
Why does this need to be exposed publicly?

Reply:
It doesn't. Good catch